app/webpacker/components/Live/components/ResultsTable.jsx

```diff
@@ -0,0 +1,172 @@
+import { Table } from 'semantic-ui-react';
+import React, { useMemo } from 'react';
+import _ from 'lodash';
+import { formatAttemptResult } from '../../../lib/wca-live/attempts';
+import { events } from '../../../lib/wca-data.js.erb';
+import { editRegistrationUrl, liveUrls } from '../../../lib/requests/routes.js.erb';
+
+const customOrderBy = (competitor, resultsByRegistrationId, sortBy) => {
+  const competitorResults = resultsByRegistrationId[competitor.id];
+
+  if (!competitorResults) {
+    return competitor.id;
+  }
+
+  return competitorResults[0][sortBy];
+};
+
+export const rankingCellStyle = (result) => {
+  if (!result) {
+    return {};
+  }
+  if (result.advancing) {
+    return { backgroundColor: 'rgb(0, 230, 118)' };
+  }
+
+  if (result.advancing_questionable) {
+    return { backgroundColor: 'rgba(0, 230, 118, 0.5)' };
+  }
+
+  return {};
+};
+
+export const recordTagStyle = (tag) => {
+  const styles = {
+    display: 'block',
+    lineHeight: 1,
+    padding: '0.3em 0.4em',
+    borderRadius: '4px',
+    fontWeight: 600,
+    fontSize: '0.6em',
+    position: 'absolute',
+    top: '0px',
+    right: '0px',
+    transform: 'translate(110%, -40%)',
+    color: 'rgb(255, 255, 255)',
+  };
+
+  switch (tag) {
+    case 'WR': {
+      styles.backgroundColor = 'rgb(244, 67, 54)';
+      break;
+    }
+    case 'CR': {
+      styles.backgroundColor = 'rgb(255, 235, 59)';
+      break;
+    }
+    case 'NR': {
+      styles.backgroundColor = 'rgb(0, 230, 118)';
+      break;
+    }
+    case 'PR': {
+      styles.backgroundColor = 'rgb(66, 66, 66)';
+      break;
+    }
+    default: {
+      return {};
+    }
+  }
+  return styles;
+};
```

Comment:
Not a fan of big custom styles at all. SemUI has a `Label` component, can that work?
For specific colors, I'll agree to some CSS styling but `display: block` and others should definitely be left to SemUI.

Reply:
yes, this was a fairly lazy way of recreating the same thing as wca live